src/battle_game/core/units/dark_fury.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include "battle_game/core/unit.h"
+#include "battle_game/core/units/tiny_tank.h"
+
+namespace battle_game::unit {
+class DarkFury : public Tank {
+ public:
+  DarkFury(GameCore *game_core, uint32_t id, uint32_t player_id);
+  void Render() override;
+  void Update() override;
+
+ protected:
+  void Fire();
+  void SwitchMode();
+  [[nodiscard]] const char *UnitName() const override;
+  [[nodiscard]] const char *Author() const override;
+
+  uint32_t fire_count_down_{0};
+  uint32_t switch_mode_count_down_{0};
+  bool is_berserk{false};
```

Comment:
use lower_case_ ending with an underscore

Reply:
It has been fixed and adjusted to conform to agreement of skills